internal/keyvalue/leveldb.go

```diff
@@ -221,19 +249,19 @@ func (db *LDBDatabase) meter(refresh time.Duration) {
 			merr = err
 			continue
 		}
-		// Find the compaction table, skip the header
+		// Find the compaction leveldbTable, skip the header
```

Comment:
Was the change from "table" to "leveldbTable" from an automated code change?

Reply:
Probably